<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="MainPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome to Babel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PasswordLineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RegisterPushButton_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>440</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordLabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>300</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="usernameLabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>241</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Log in or register to a new account</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoginpushButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>390</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoginlineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RegisterPage">
     <widget class="QLabel" name="RegisterPage_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="Id_label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="Pwd_label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="PwdConflabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Password confirmation</string>
      </property>
     </widget>
     <widget class="QLabel" name="Bd_label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>340</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Birth date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_lineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pwd_lineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="PwdConf_lineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>300</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="Bd_dateEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>360</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BackLogin_pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>430</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to log in</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome !</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Call_id_lineEdit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter contact Id  to start a call</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Call_pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Call</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LogOut_pushButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>270</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Call started</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>End call</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="AcceptCall_pushButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DenyCall_pushButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Deny</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Someone is calling you</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
